tests/scalars/CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
-file(GLOB test_cases_list *.cpp)
+
+if(SYCL_CTS_ENABLE_OPENCL_INTEROP_TESTS)
+  file(GLOB test_cases_list *.cpp)
+else()
+  file(GLOB test_cases_list scalars_sycl_types.cpp)
+endif()
```

Comment:
I think it's better to apply the opposite strategy and explicitly remove source files with OpenCL interop tests. E.g. https://github.com/KhronosGroup/SYCL-CTS/blob/62155afca2514a208324a9df5d8fd39eddb3caee/util/CMakeLists.txt#L4-L6.

Reply:
Done